pkg/controller/egressgateway/egressgateway_controller.go

```diff
@@ -186,6 +205,28 @@ func (r *ReconcileEgressGateway) Reconcile(ctx context.Context, request reconcil
 			reqLogger.Info("EgressGateway object not found")
 			// Since the EGW resource is not found, remove the deployment.
 			r.status.RemoveDeployments(types.NamespacedName{Name: request.Name, Namespace: request.Namespace})
+
+			// In case of openShift, we are using a single SCC.
```

Comment:
nit: "In the case of OpenShift" (add "the", and also capitalized since it's a name)